taxi/urls.py

```diff
@@ -20,7 +26,31 @@
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path("car/create/", CarCreateView.as_view(), name="car-create"),
+    path(
+        "car/<int:pk>/delete/",
+        CarDeleteView.as_view(),
+        name="car-delete"
+    ),
+    path("car/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
+    path(
+        "manufacturer/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path(
+        "manufacturer/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturer/<int:pk>/delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete"
     ),
```

Comment:
we use plural form of urls.
cars/
cars/{id}/
cars/{id}/info/
cars/only_expensive/